Spanner/tests/Unit/DatabaseTest.php

```diff
@@ -129,6 +130,20 @@ public function set_up()
         ];
 
         $this->database = TestHelpers::stub(Database::class, $args, $props);
+
+        $args = [
+            $this->connection->reveal(),
+            $this->instance,
+            $this->lro->reveal(),
+            $this->lroCallables,
+            self::PROJECT,
+            self::DATABASE,
+            null,
+            false,
+            [],
+            'Reader'
```

Comment:
Better to append the original `$args` array with the new values than re-init?

Reply:
That should work. Changing now.